config/hocon-mp/src/main/java/io/helidon/config/hocon/mp/HoconMpConfigIncluder.java

```diff
@@ -59,7 +59,6 @@ public ConfigIncluder withFallback(ConfigIncluder fallback) {
 
     @Override
     public ConfigObject include(ConfigIncludeContext context, String what) {
-        new Exception().printStackTrace();
```

Comment:
This exception looks a little strange. It was [since the class was created](https://github.com/helidon-io/helidon/commit/bbf31fa6b150996e29a0601c8055be25f1d0c40c#diff-6cbd81fad07d03e1b55693b174985f5ace733140ed24ef2da4490da3a20126d8R62).

Reply:
Is this exception needed for something, right?

Reply:
That seems to be some debug statement that stayed in production code.

Reply:
Definitely, a bad oversight. I agree with his change.